examples/bls/extra/usr/lib/initcpio/hooks/composefs

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/ash
+# SPDX-License-Identifier: GPL-2.0-only
+
+run_latehook() {
+    local composefs
+
+    composefs="$(getarg composefs)"
+    if [ -z "$composefs" ]; then
+        return 0
+    fi
+
+    /usr/bin/composefs-pivot-sysroot /new_root
+}
+
+# vim: set ft=sh ts=4 sw=4 et:
```

Comment:
Can you drop that?  And maybe the licence?  The amount of stuff you "copied" here is not really copyrightable.